tests/constants.py

```diff
@@ -442,9 +452,6 @@ def test_inf_of_math(self):
   def test_nan_of_math(self):
     self.assertOnlyIn((3, 5), self.detect("import math\nmath.nan"))
 
-  def test_name_of_contextvars_ContextVar(self):
-    self.assertOnlyIn((3, 7), self.detect("from contextvars import ContextVar\nContextVar.name"))
-
```

Comment:
Why is this one removed?

Reply:
Because the `contextvars` module itself is already 3.7+ (when not backported). In the backport version, `contextvars.ContextVar.name` exists. This rule might be due to [the "New in version 3.7.1" marker in the documentation](https://docs.python.org/3/library/contextvars.html#contextvars.ContextVar.name), but since `vermin` only handles major and minor versions (no patch version), we only write this as `(3, 7)`, and then this rule becomes unnecessary.

Reply:
Good point 👍🏻